Improve error message after demo

1. When job fails tell user to go hit the button next to the job instead of providing a link.
2. When report fails improve message. for example:
    [ERROR] The submitted chart has failed verification. Reason(s): kubeversion is not specified.
     Please run the (link-to-verifier-tool) and ensure all checks pass.
3. Improve the link to http://bit.ly/technology-partner-success-desk

--- a/scripts/prepare_pr_comment.py
+++ b/scripts/prepare_pr_comment.py
@@ -2,29 +2,29 @@ import os
 import sys
 
 def prepare_failure_comment(repository, issue_number, vendor_label, chart_name):
-    runid = open("./pr/build-verify-check-run-id").read()
-    run_url = f"https://github.com/{repository}/pull/{issue_number}/checks?check_run_id={runid}".strip()
     msg = f"""\
 Thank you for the pull request #{issue_number}!
 
 There are few errors while building and verifying your pull request.
-Please check the error log for more details:
-{run_url}
-(open this link in a new tab/window)
+To see the console output with the error messages, click the "Details"
+link next to "Build and Verify" job status towards the end of this page.
 """
     if os.path.exists("./pr/errors"):
         errors = open("./pr/errors").read()
         msg += f"""
-The following issues require changes in the pull request:
+[ERROR] The submitted chart has failed verification. Reason(s):
 
 {errors}
+
+Please run the [chart-verifier](https://github.com/redhat-certification/chart-verifier) \
+and ensure all checks pass.
 """
 
     msg += f"""
 ---
 /metadata {{"vendor_label": "{vendor_label}", "chart_name": "{chart_name}"}}
 
-Partner Support: http://bit.ly/technology-partner-success-desk
+For support, connect with our [Technology Partner Success Desk](https://redhat-connect.gitbook.io/red-hat-partner-connect-general-guide/managing-your-account/getting-help/technology-partner-success-desk).
 """
     return msg
 

--- a/scripts/prepare_pr_comment.py
+++ b/scripts/prepare_pr_comment.py
@@ -17,7 +17,7 @@ link next to "Build and Verify" job status towards the end of this page.
 {errors}
 
 Please run the [chart-verifier](https://github.com/redhat-certification/chart-verifier) \
-and ensure all checks pass.
+and ensure all mandatory checks pass.
 """
 
     msg += f"""

